index.html

```diff
@@ -0,0 +1,285 @@
+                            <i class="fas fa-mobile-alt fa-4x"></i>
+                            <h3>Mobile Development</h3>
+                            <p>
+                                The world is living in a mobile trend nowadays. Learn how to build
+                                mobile apps similar to WhatsApp, Facebook...
+                            </p>
+                        </div>
+                    </li>
+                </ul>
+                <button class="join_summit">Join Code 4 Easy Summit 2022</button>
+            </div>
+        </section>
+        <section class="speakers container">
+            <div class="title-div">
+                <p class="title">Featured Speakers</p>
+                <hr class="red-line">
+            </div>
+            <ul class="list">
+                <li>
+                    <div class="card">
+                        <div class="profile">
+                            <img src="./assets/images/back.png" alt="back" class="check">
+                            <img src="./assets/images/Ishmael.jpeg" alt="profile" class="avater">
+                        </div>
+                        <div class="details">
+                            <h3>Ishmael Kargbo</h3>
+                            <p class="position">Student at Microvers</p>
+                            <hr />
+                            <p class="about">
+                                Lorem ipsum dolor sit amet consectetur adipisicing elit. Repudiandae pariatur aut atque
+                                consequatur corporis ullam perferendis, soluta necessitatibus unde earum suscipit odit.
+                                Vel
+                                deserunt iste repellat fugit nostrum quaerat accusamus.
+                            </p>
+                        </div>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <div class="profile">
+                            <img src="./assets/images/back.png" alt="back" class="check">
+                            <img src="./assets/images/Ishmael.jpeg" alt="profile" class="avater">
+                        </div>
+                        <div class="details">
+                            <h3>Ishmael Kargbo</h3>
+                            <p class="position">Student at Microvers</p>
+                            <hr />
+                            <p class="about">
+                                Lorem ipsum dolor sit amet consectetur adipisicing elit. Repudiandae pariatur aut atque
+                                consequatur corporis ullam perferendis, soluta necessitatibus unde earum suscipit odit.
+                                Vel
+                                deserunt iste repellat fugit nostrum quaerat accusamus.
+                            </p>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </section>
```

Comment:
This Feature Speakers section needs to be rendered dynamically by JavaScript in order to respect the project requirements. 
![image](https://user-images.githubusercontent.com/83596135/163474037-2f87821b-7788-484d-882b-6f5ba2138dfa.png)


- [ ] Kindly, implement the required functionality using JavaScript;
- [ ] Please take care to display 6 different speakers (you need an array of objects with each speaker and its details stored there; You should loop thru that array and display on the UI the needed information);

Reply:
Ok 👍